apps/state_mediator/test/state_mediator/integration/gtfs_test.exs

```diff
@@ -416,7 +416,7 @@ defmodule StateMediator.Integration.GtfsTest do
     test "have valid route, trip and stop IDs" do
       for %{route_id: route_id} <- all_routes(),
           prediction <-
-            State.Prediction.select_grouped([State.Prediction], [
+            State.Prediction.select([
```

Comment:
How was this test passing before it was removed previously?

Reply:
We aren't running it in CI :/ It only runs if we provide a source for predictions in the integration tests, and we aren't doing that at the moment. Once it lands I can add it back.